cfme/tests/control/test_basic.py

```diff
@@ -694,3 +730,28 @@ def test_control_is_ansible_playbook_available_in_actions_dropdown(action_collec
     """
     view = navigate_to(action_collection, "Add")
     assert "Run Ansible Playbook" in [option.text for option in view.action_type.all_options]
+
+
+@pytest.mark.tier(2)
+def test_alerts_monitor_overview_page(alert_collection, virtualcenter_provider,
+        setup_for_monitor_alerts):
+    """
+    Polarion:
+        assignee: jdupuy
+        casecomponent: control
+        initialEstimate: 1/12h
+    """
```

Comment:
@john-dupuy 
Please add  ```testSteps:``` and ```caseimportance:``` under polarion details.

Reply:
@ganeshhubale `testSteps` are only necessary for manual test cases, correct?

Reply:
@ganeshhubale: Yup, `stestSteps` is `Polarion` tag and its optional for `Automated` as we have that tradition. My observation: when, we transfer `FA` to new owner its very difficult to understand failed automated test steps as we have to test it manually; But, @john-dupuy added steps in comment of code :+1: So this must be contributor choice means optional suggestion.

Reply:
Thanks for clearing my doubt :+1: